02 Writing Algorithms/28 Indicators/01 Supported Indicators/77 Target Downside Deviation/03 Update Manual Indicators.html

```diff
@@ -11,7 +11,8 @@ <h4>Automatic Update</h4>
 private TargetDownsideDeviation _tdd;
 
 // In Initialize()
-_tdd = new TargetDownsideDeviation(name, period);
+_roc = new RateOfChange(1);
+_tdd = (new TargetDownsideDeviation(period)).Of(_roc);
 _tdd.Updated += IndicatorUpdateMethod;
 
 RegisterIndicator(symbol, _tdd, Resolution.Daily);
```

Comment:
Double-check this. I don't think it actually works.
https://github.com/QuantConnect/Lean/issues/6708

I think we need to pass `_roc` instead of `_tdd`